<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>461</width>
      <height>241</height>
     </rect>
    </property>
    <property name="title">
     <string>日期时间</string>
    </property>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>461</width>
       <height>231</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QPushButton" name="getDateTimeBtn">
        <property name="text">
         <string>读取当前日期时间</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>         时 间</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="timeLineEdit"/>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="settingTimeBtn">
        <property name="text">
         <string>设置时间</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>        日 期</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLineEdit" name="dateLineEdit"/>
      </item>
      <item row="2" column="3">
       <widget class="QPushButton" name="settingDateBtn">
        <property name="text">
         <string>设置日期</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>      日期时间</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QLineEdit" name="dateTimeLineEdit"/>
      </item>
      <item row="3" column="3">
       <widget class="QPushButton" name="settingDateTimeBtn">
        <property name="text">
         <string>设置日期时间</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>                                  </string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>            字符串</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QTimeEdit" name="timeEdit"/>
      </item>
      <item row="2" column="1">
       <widget class="QDateEdit" name="dateEdit"/>
      </item>
      <item row="3" column="1">
       <widget class="QDateTimeEdit" name="dateTimeEdit"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>30</y>
      <width>291</width>
      <height>351</height>
     </rect>
    </property>
    <property name="title">
     <string>日期选择</string>
    </property>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>选择日期：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="showDateLineEdit">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>30</y>
       <width>191</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QCalendarWidget" name="calendarWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>271</width>
       <height>241</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>290</y>
      <width>461</width>
      <height>241</height>
     </rect>
    </property>
    <property name="title">
     <string>定时器</string>
    </property>
    <widget class="QPushButton" name="timerStartBtn">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>开始</string>
     </property>
    </widget>
    <widget class="QPushButton" name="timeStopBtn">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>50</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>停止</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>定时周期</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="timerCircleSpinBox">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>100</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="settingCircleBtn">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>100</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>设置周期</string>
     </property>
    </widget>
    <widget class="QLabel" name="elapsTimeLab">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>50</y>
       <width>53</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>时间流逝</string>
     </property>
    </widget>
    <widget class="QLCDNumber" name="hourLcdNumber">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>150</y>
       <width>71</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QLCDNumber" name="minLcdNumber">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>150</y>
       <width>71</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QLCDNumber" name="secLcdNumber_3">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>150</y>
       <width>71</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QProgressBar" name="progressBar">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>200</y>
       <width>271</width>
       <height>23</height>
      </rect>
     </property>
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>450</y>
      <width>231</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>退出</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
